common/selectors/colorSelectors.js

```diff
@@ -60,11 +60,19 @@ const selectSiteIcon = createSelector(
     const backgroundColor = toRGBString(backgroundRGB);
     const fontColor = getBlackOrWhite(...backgroundRGB);
     const firstLetter = prettyUrl(parsedUrl.hostname)[0];
+
+    const almostWhiteThreshold = 230;
+    let backgroundColorIsAlmostWhite = false;
+    if (backgroundRGB[0] > almostWhiteThreshold && backgroundRGB[1] > almostWhiteThreshold && backgroundRGB[2] > almostWhiteThreshold) {
+      backgroundColorIsAlmostWhite = true;
+    }
+
```

Comment:
I'll add a todo to add testing and maybe make it more generic. this is pretty hacky